src/hevm/src/EVM/Format.hs

```diff
@@ -213,20 +235,32 @@ showTrace dapp trace =
       t
     FrameTrace (CreationContext hash _ _ ) ->
       "create " <> maybeContractName (preview (dappSolcByHash . ix hash . _2) dapp) <> pos
-    FrameTrace (CallContext _ _ hash abi calldata _ _) ->
-      case preview (dappSolcByHash . ix hash . _2) dapp of
+    FrameTrace (CallContext target context _ _ hash abi calldata _ _) ->
+      let calltype = if (target == context)
+                     then "call "
+                     else "delegatecall "
```

Comment:
this could also be `CALLCODE` I believe

Reply:
Not as simple to determine unfortunately, so it will just show up as delegatecall.

Reply:
should we display it as `delegatecall/callcode`?

Reply:
Imo no because it's clutter the vast majority of the time. Does anyone use callcode anymore?

Reply:
If we really wanted to show it we could add a new `calltype` field to `CallContext`.

Reply:
If you still think it's worth it, then how about also distinguishing `create` and `create2`?

Reply:
ok, I'm fine with `delegatecall` for now

Reply:
We also don't distinguish `staticcall`.